remsfal-core/src/main/java/de/remsfal/core/model/AddressModel.java

```diff
@@ -1,10 +1,15 @@
 package de.remsfal.core.model;
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import org.immutables.value.Value;
+
 import java.util.Locale;
 
 /**
  * @author Alexander Stanik [alexander.stanik@htw-berlin.de]
  */
+@Value.Immutable
```

Comment:
A model is just an interface that defines a common language between our boundary and the entity layer. Therefore, all Immutables and Jackson annotation should only be used in the json package.